.travis.yml

```diff
@@ -44,8 +44,9 @@ jobs:
         # in a GitHub Workflow.
         - make unit pkg=... case=_NONE_
 
-        # Step 3: Lint go code. Limit to 1 worker to reduce memory usage.
-        - make lint workers=1
+        # Step 3: Lint go code. Limit to 1 worker and invoke GC more often to
+        # reduce memory usage.
+        - GOGC=20 make lint workers=1
```

Comment:
Would it OOM on travis because of the increased number of lines to lint?

Reply:
Yes. And if I look at GitHub Actions, it uses more than 8GB of RAM there when not restricted. As the lint on Travis only acts as a gatekeeper to prevent non-linted code to block our build queues, maybe it would make sense to exclude the `itest` directory there to speed things up? Or maybe there's a way to just include lines changed in the PR? I'll investigate.

Reply:
Okay, restricting only to the commits of a PR doesn't speed things up. The linter will still analyze everything but only _report_ problems added in the specified commits. I think excluding some directories might be the way to go. Or we just accept the 3 minutes this takes before each Travis run.

Reply:
Yeah, I think that is okay. I would be interested to see if we could increase the number of workers on Travis though, as that would most likely be the way to speed it up (if it doesn't OOM)

Reply:
Ah, good idea. If we can limit the amount of memory used through the environment variables, we should be able to increase the worker count again.
I looked it up on the Travis spec page and apparently the VMs have 7.5GB. I'm trying with `GTOGC=50` and no worker limit which resulted in 6.3GB being used on my machine. Let's see if it OOMs with that on Travis or not.

Reply:
Looking good:
```
$ GOGC=50 make lint
...
INFO Memory: 396 samples, avg is 3384.2MB, max is 4855.0MB 
INFO Execution took 1m40.098082206s   
```